strapi/strapi
# [TS] Fix static resources resolution

### What does it do?

Set some resources as static & don't copy them into the dist folder as it doesn't make sense (public folder, package.json, favicon.ico, etc...)

### Why is it needed?

Copying folders that are meant to be static can end up in storage/performances issues (duplicating the public/uploads folder for instance) & sometimes just don't make sense (package.json)

### How to test it?

- Start the kitchensink-ts app both with develop & build/start and make sure we're still able to use the upload plugin.
